editor/scene_tree_editor.cpp

```diff
@@ -386,10 +386,18 @@ void SceneTreeEditor::_add_nodes(Node *p_node, TreeItem *p_parent) {
 
 		Ref<Script> script = p_node->get_script();
 		if (!script.is_null()) {
-			item->add_button(0, get_theme_icon(SNAME("Script"), SNAME("EditorIcons")), BUTTON_SCRIPT, false, TTR("Open Script:") + " " + script->get_path());
+			String additional_notes;
+			// Can't set tooltip after adding button, need to do it before.
+			if (script->is_tool()) {
+				additional_notes += TTR("\nThis script is currently running in the editor.");
```

Comment:
The `\n` shouldn't be part of the `TTR()`, it shouldn't be the translators' responsibility to start their translation with an empty line.

Reply:
Done.